Gruntfile.js

```diff
@@ -586,9 +587,9 @@ module.exports = function(grunt) {
   });
 
   // Build tasks
-  grunt.registerTask('mmnpm', 'Update and patch npm dependencies', [
+  grunt.registerTask('mmyarn', 'Update and patch dependencies', [
```

Comment:
Maybe we should rename this to something generic and more descriptive, eg: `install_dependencies`?

Reply:
Yep makes sense